libbeat/tests/integration/template_test.go

```diff
@@ -294,13 +295,16 @@ logging:
 	mockbeat := NewBeat(t, "mockbeat", "../../libbeat.test")
 	mockbeat.WriteConfigFile(cfg)
 	mockbeat.Start()
-	mockbeat.WaitForLogs("mockbeat start running.", 60*time.Second)
+	mockbeat.WaitLogsContains("mockbeat start running.", 60*time.Second)
 	require.Eventually(t, func() bool {
 		return mockbeat.LogMatch("doBulkRequest: [[:digit:]]+ events have been sent")
 	}, 20*time.Second, 100*time.Millisecond, "looking for PublishEvents")
 
 	u := fmt.Sprintf("%s/_index_template/%s", esUrl.String(), datastream)
-	r, _ := http.Get(u)
+	r, err := http.Get(u) //nolint:gosec,noctx // It's a test
+	if err != nil {
+		defer r.Body.Close()
+	}
```

Comment:
if you're catching the error, could you also `require` it's nil before proceeding?